contracts/interfaces/validator/ISlashingExecution.sol

```diff
@@ -27,7 +27,12 @@ interface ISlashingExecution {
    * Emits the event `ValidatorPunished`.
    *
    */
-  function execSlash(address cid, uint256 newJailedUntil, uint256 slashAmount, bool cannotBailout) external;
+  function execSlash(
+    address cid,
+    uint256 newJailedUntil,
+    uint256 slashAmount,
+    bool cannotBailout
+  ) external returns (bool slashed);
```

Comment:
Why we need return this `slashed` here?

Reply:
will revert this changes: https://sky-mavis.slack.com/archives/C05DYV21A72/p1713951395869659?thread_ts=1713950652.985619&cid=C05DYV21A72